modules/nf-core/kmcp/index/main.nf

```diff
@@ -0,0 +1,47 @@
+process KMCP_INDEX {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::kmcp=0.9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kmcp:0.9.1--h9ee0642_0':
+        'quay.io/biocontainers/kmcp:0.9.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(compute_output)
+
+    output:
+    tuple val(meta), path("${prefix}")   , emit: kmcp
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    kmcp \\
+        index \\
+        --in-dir $compute_output \\
+        $args \\
+        --threads $task.cpus \\
+        --out-dir ${prefix}
```

Comment:
IT might be useful to by default generate and emit in `out:` the log file with `--log ${prefix}.log` (thinking MultiQC or something ;) )